lib/chef/provider/package/apt.rb

```diff
@@ -71,15 +71,11 @@ def candidate_version
         end
 
         def package_locked(name, version)
-          islocked = false
           locked = shell_out_compact_timeout!("apt-mark", "showhold")
-          locked.stdout.each_line do |line|
-            line_package = line.strip
-            if line_package == name
-              islocked = true
-            end
+          locked_packages = locked.stdout.each_line.map do |line|
```

Comment:
A fun Ruby shorthand for this is:

```ruby
locked_packages = locked.stdout.each_line.map(&:strip)
```

You don't have to change it - just want to impart knowledge I've learned from much smarter people than me. :)